nomenclature/definitions/region/subcountries.yaml

```diff
@@ -275,33 +719,81 @@ Italy|Subgrid south:
     ehighway2050: 112_NS
     plan4res: NSDE
 
 The Netherlands|North Sea:
-    definition: this 'virtual' region holds the offshore windpower of the netherlands in norh sea
+    definition: This 'virtual' region represents the offshore windpower of the netherlands in Norh Sea
     country: The Netherlands
     notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
     ehighway2050: 111_NS
     plan4res: NSNL
 
 United Kingdom|North Sea:
-    definition: this 'virtual' region holds the offshore windpower of the UK in norh sea
+    definition: This 'virtual' region represents the offshore windpower of the UK in Norh Sea
     country: United Kingdom
     notes: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions
     ehighway2050: 106_NS+107_NS+108_NS+109_NS
     plan4res: NSUK
-    
+
+United Kingdom|North Sea|Subgrid 106:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to the south of UK
+    country: United Kingdom
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 106_NS
+
+United Kingdom|North Sea|Subgrid 107:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to the centre-south of UK
+    country: United Kingdom
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 107_NS
+
+United Kingdom|North Sea|Subgrid 108:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to the centre-north of UK
+    country: United Kingdom
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 108_NS
+
+United Kingdom|North Sea|Subgrid 109:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to the north of UK
+    country: United Kingdom
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 109_NS
+
+Denmark|North Sea|Subgrid 113:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to Denmark
+    country: Denmark
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 113_NS
+
+Denmark|North Sea|Subgrid 114:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to Denmark
+    country: Denmark
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 114_NS
+
+Norway|North Sea|Subgrid 115:
+    definition: This 'virtual' region represents the offshore windpower in the Norh Sea near to Norway
+    country: Norway
+    notes: This region was defined and used in the ehighway2050 public datasets
+    ehighway2050: 115_NS
+
+Sweden|North Sea|Subgrid 116:
```

Comment:
Wouldn't that be the Baltic Sea?

Reply:
This region is located at (58°20'04.2"N 10°07'15.2"E). So, it is between Northern Denmark, Southern Norway, and Western Sweden in Map. Thus, I assigned it to Sweden to avoid multiple of these regions assigned to only one country.

Reply:
ok, thanks for double-checking